librosa/core/audio.py

```diff
@@ -527,22 +527,27 @@ def resample(
             `soxr` Very high-, High-, Medium-, Low-quality FFT-based bandlimited interpolation.
             ``'soxr_hq'`` is the default setting of `soxr`.
         'soxr_qq'
-            `soxr` Quick cubic interpolation (very fast)
+            `soxr` Quick cubic interpolation (very fast, but not bandlimited)
         'kaiser_best'
             `resampy` high-quality mode
         'kaiser_fast'
             `resampy` faster method
         'fft' or 'scipy'
             `scipy.signal.resample` Fourier method.
         'polyphase'
-            `scipy.signal.resample_poly` polyphase filtering. (fast)
+            `scipy.signal.resample_poly` polyphase filtering. (fast, but not bandlimited)
```

Comment:
actually, not sure this is correct?  The scipy documentation explicitly states that the signal is low-pass filtered: https://docs.scipy.org/doc/scipy/reference/generated/scipy.signal.resample_poly.html

Reply:
Might be, but it doesn't look like a good one.

<img width="401" alt="image" src="https://user-images.githubusercontent.com/1258637/205642032-87df8985-4f36-44df-ac75-08040bba890b.png">

Reply:
Ref my colab: https://colab.research.google.com/drive/1VNS8O494CYocHrilUc7lZ_hKltNqb_PG?usp=sharing#scrollTo=4D3_oaecydbH

Reply:
Oh indeed, it's not a *good* filter, but it's qualitatively different from something like zero-order hold.

Reply:
Yes. With linear you get a clean aliased pyramid :-)

<img width="400" alt="image" src="https://user-images.githubusercontent.com/1258637/205642978-e13e1950-a257-4916-8844-f5333f25ed44.png">

Reply:
Should I remove the `but not bandlimited` completly for `polyphase`, or change it to something like `slightly bandlimited`?

Reply:
It will still be covered by the `note` section.